discord/enums.py

```diff
@@ -74,6 +74,13 @@ class VerificationLevel(Enum):
     def __str__(self):
         return self.name
 
+class DefaultMessageNotifications(Enum):
```

Comment:
I am not entirely sold on this name.

It's too verbose.

`default_message_notifications=DefaultMessageNotifications.only_mentions` is quite a mouthful. I know there's `verification_level` but even that one is pushing it.